functions/venue.js

```diff
@@ -648,8 +648,6 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
 
   if (data.mapBackgroundImageUrl) {
     updated.mapBackgroundImageUrl = data.mapBackgroundImageUrl;
-  } else {
-    updated.mapBackgroundImageUrl = "";
   }
```

Comment:
Do we have any features currently that want to be able to delete the `mapBackgroundImageUrl`? Or will it always just be replaced with a new url; never deleted/set back to default?

Reply:
We do not have any features that delete the mapBackgroundImageUrl. This update functions updates other venue information and should always persist the background image.

Reply:
Awesome, thanks for the context.